Xamarin.Forms.Platform.UAP/CellControl.cs

```diff
@@ -103,9 +103,10 @@ protected override Windows.Foundation.Size MeasureOverride(Windows.Foundation.Si
 			if (lv != null)
 			{
 				lv.SetValue(MeasuredEstimateProperty, result.Height);
-				SetDafaultColor();
 			}
 
+			SetDefaultSwitchColor();
```

Comment:
Can’t say about the listview, maybe the original author didn’t realize it could also be contained in a table view. As for the measuring, have a look at #4883. You can’t set the color until the control is fully loaded. In the said pr, I did it by using the OnLoaded event, the author of this probably found a similar point in the lifetime cycle by using the measure event